index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Nemwel Boniface">
+    <meta name="description" content="This is my personal portfolio site that I will share to all potential employers to showcase my skills as a developer">
+
+    <!--google Fonts roboto, crete-round-->
+    
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+
+    <!-- font awesome icons -->
+
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Nemwel Portfolio</title>
+
+    <!--Page stylesheets-->
+    <link rel="stylesheet" href="stylesheets/landing.css">
+
+    <!--Smooth scroll script-->
+       
+    <script src="https://cdn.jsdelivr.net/gh/cferdinandi/smooth-scroll/dist/smooth-scroll.polyfills.min.js"></script>
+
+</head>
+<body>
+    <header class="header">
+        <div class="topHeaderBar">
+            <div class="time-left">
+                <h2>9:41</h2>
+            </div>
+
+            <div class="icons-right">
+                <i class="fa fa-signal" aria-hidden="true"></i>
+                <i class="fa fa-wifi" aria-hidden="true"></i>
+                <i class="fa fa-battery-full" aria-hidden="true"></i> 
+            </div>
+        </div>
+
+        <nav class="topnavBar">
+            <div class="navleft">
+                <h2>Nemwel Boniface</h2>
+            </div>
+
+            <div class="navright">
+                <i class="fa fa-bars" aria-hidden="true"></i>
+            </div>
+        </nav>
+    </header>
+
+    <main class="main">
+        <article class="authordetails">
+            <div class="authorheader">
+                <h2>Hey There. I'm Nemwel</h2>
```

Comment:
- [ ] The `<h2>` on line 46 should be font-weight bold (or 700).  Let's change it, so the project looks a lot more like the Figma design given.

Reply:
Hello @Meltrust  Thank you for this review.
- I have changed the font weight for the h2 element and made it to 700 as suggested.
- I have changed the container headline from article to section as suggested
- I however am having trouble placing the main background image to the bottom of the page. Kindly assist me with this bit.

Reply:
Hey Nemwel,

As a suggestion, you could use `position-absolute`, `top`, `bottom`, `right`, and `left` in the CSS in order to position an element exactly where you want it like this:

```CSS
.element {
position: absolute;
top; 12%;
left: 50px;
right: 0;
bottom: 5%;
}
```
This is just an example to give you an idea ☝️ 
Try to get familiar with `position: relative` and `position: absolute` and even `position: sticky` as early as you can, because those positioning properties are really useful!

Kindly check [this](https://www.w3schools.com/cssref/pr_class_position.asp) and pay special attention to the section "Property values" and check the examples.

In case you are still blocked you can go [here](https://questions.microverse.org/) it's really helpful 👍.

Reply:
Hello @Meltrust Thank you for this tip. let me work on the changes